src/Microsoft.DotNet.XHarness.CLI/Commands/WASM/Browser/WasmTestBrowserCommand.cs

```diff
@@ -105,8 +109,37 @@ protected override async Task<ExitCode> InvokeInternal(ILogger logger)
         }
 
         private (DriverService, IWebDriver) GetChromeDriver(ILogger logger)
+            => GetChromiumDriver<ChromeOptions, ChromeDriver, ChromeDriverService>(
+                        "chromedriver",
+                        options => ChromeDriverService.CreateDefaultService(),
+                        logger);
+
+        private (DriverService, IWebDriver) GetEdgeDriver(ILogger logger)
+        {
+            string driverName = "msedgedriver";
+            if (RuntimeInformation.IsOSPlatform(OSPlatform.Windows))
+                driverName = "msedgedriver.exe";
+
+            string? driverPath = Environment.GetEnvironmentVariable("PATH")
+                                    ?.Split(':')
```

Comment:
PATH is split by semicolon on Windows, should be changed to:

```suggestion
                                    ?.Split(Path.PathSeparator)
```